controller/work_item_link.go

```diff
@@ -314,31 +296,37 @@ func (c *WorkItemLinkController) Delete(ctx *app.DeleteWorkItemLinkContext) erro
 	if !authorized {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("user is not authorized to delete the link"))
 	}
-	return application.Transactional(c.db, func(appl application.Application) error {
-		linkCtx := newWorkItemLinkContext(ctx.Context, ctx.Service, appl, c.db, ctx.Request, ctx.ResponseWriter, app.WorkItemLinkHref, currentUserIdentityID)
-		return deleteWorkItemLink(linkCtx, ctx, ctx.LinkID)
+	err = application.Transactional(c.db, func(appl application.Application) error {
+		return appl.WorkItemLinks().Delete(ctx.Context, ctx.LinkID, *currentUserIdentityID)
 	})
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+	return ctx.OK([]byte{})
 }
 
 // Show runs the show action.
 func (c *WorkItemLinkController) Show(ctx *app.ShowWorkItemLinkContext) error {
-	return application.Transactional(c.db, func(appl application.Application) error {
-		modelLink, err := appl.WorkItemLinks().Load(ctx.Context, ctx.LinkID)
-		if err != nil {
-			return jsonapi.JSONErrorResponse(ctx, err)
+	var modelLink *link.WorkItemLink
+	err := application.Transactional(c.db, func(appl application.Application) error {
+		var err error
+		modelLink, err = appl.WorkItemLinks().Load(ctx.Context, ctx.LinkID)
+		return err
+	})
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+	return ctx.ConditionalRequest(*modelLink, c.config.GetCacheControlWorkItemLink, func() error {
+		// convert to rest representation
+		appLink := ConvertLinkFromModel(ctx.Request, *modelLink)
+		if err := enrichLinkSingle(ctx.Context, c.db, ctx.Request, &appLink); err != nil {
+			return err
 		}
-		return ctx.ConditionalRequest(*modelLink, c.config.GetCacheControlWorkItemLink, func() error {
-			// convert to rest representation
-			appLink := ConvertLinkFromModel(ctx.Request, *modelLink)
-			if err := enrichLinkSingle(ctx.Context, appl, ctx.Request, &appLink); err != nil {
-				return jsonapi.JSONErrorResponse(ctx, err)
-			}
-			return ctx.OK(&appLink)
-		})
+		return ctx.OK(&appLink)
 	})
 }
 
-// ConvertLinkFromModel converts a work item from model to REST representation
```

Comment:
comment on exported function ConvertLinkToModel should be of the form "ConvertLinkToModel ..."